### DISCUSSION TITLE
Is there a clear milestone of Netty5 release date?

### DISCUSSION THREAD

**Discussion by microhardsmith** [netty/netty#13079] [General]
Haven't noticed a official release date about Netty5

**Comment by chrisvest** [comment]
There is no official timeline for the release, no.

**Comment by stanobert** [comment]
Could you please provide a timeline for Netty5 as the host name validation vulnerability is creating security challenges..

**Reply by chrisvest** [reply]
That CVE is being disputed, so security scanners should stop flagging it at some point.

**Reply by ledigiacomo** [reply]
Its been my experience that its been incredibly difficult to get disputed CVEs un-flagged by most security tooling (BlackDuck, Prisma, etc). Instead they will usually continue to flag it and add a note that the CVE is being disputed. Its not your problem of course (nor should it be), but many large companies have security policies that care only about the CVEs and not any notes in them. If its possible a release or goal date, even if its vague, for releasing Netty 5 can help with those policies alot :) .

Thanks for your time and all you do on this project!
